ConnectorRevit/ConnectorRevit/Storage/StreamStateManager.cs

```diff
@@ -36,8 +43,10 @@ public static List<StreamState> ReadState(Document doc)
 
       return states;
     }
-    catch (Exception e)
+    catch (Exception ex)
```

Comment:
Let's use the !IsFatal(ex) pattern and remove the TODO we will see what exception turn up as we logging it